marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/Utxo/UtxoIndex.hs

```diff
@@ -457,23 +459,33 @@ propUtxoQueryByAddressAndSlotInterval = property $ do
             raiseException $
               Utxo.open ":memory:" (Utxo.Depth 1) False
                 >>= Storable.insertMany events
-        let
+        let {-
```

Comment:
@berewt fourmolu insists on moving the comments through out this section of the code, a lambda function, to this location.

Reply:
I think the issue might me emacs related and related to this variable I set to use fourmolu:
`lsp-haskell-formatting-provider`